app/src/main/java/org/eyeseetea/malariacare/domain/usecase/push/PushUseCase.java

```diff
@@ -41,6 +35,8 @@
         void onApiCallError();
 
         void onApiCallError(ApiCallException e);
+
+        void onNullContext();
```

Comment:
UseCase should invoke callbacks with domain concepts in name, null and context are not domain concepts